rm/rm-node/src/main/java/org/ow2/proactive/resourcemanager/utils/RMNodeStarter.java

```diff
@@ -681,6 +681,9 @@ private void configureProActiveDefaultConfigurationFile() {
             if (defaultProActiveConfiguration.exists()) {
                 System.setProperty(CentralPAPropertyRepository.PA_CONFIGURATION_FILE.getName(),
                                    defaultProActiveConfiguration.getAbsolutePath());
+            } else {
+                System.setProperty(CentralPAPropertyRepository.PA_CONFIGURATION_FILE.getName(),
+                                   RMNodeStarter.class.getResource("/config/network/node.ini").toString());
```

Comment:
can you create a default static final string to hold this value : "/config/network/node.ini"   ?